libs/jump_portrait/src/jump_portrait/utils.py

```diff
@@ -72,7 +75,7 @@ def parallel(
         jobs = len(iterable)
     slices = slice_iterable(iterable, jobs)
     result = Parallel(n_jobs=jobs, timeout=timeout)(
-        delayed(func)(chunk, idx, *args, **kwargs)
+        delayed(func)(chunk, idx, print_progress, *args, **kwargs)
```

Comment:
print_progress is a bit too verbose. Let us rename it to "verbose" to follow conventions.